spec/electra/executionpayload_ssz.go

```diff
@@ -436,7 +436,7 @@ func (e *ExecutionPayload) HashTreeRootWith(hh ssz.HashWalker) (err error) {
 				hh.MerkleizeWithMixin(elemIndx, byteLen, (1073741824+31)/32)
 			}
 		}
-		hh.MerkleizeWithMixin(subIndx, num, 1073741824)
+		hh.MerkleizeWithMixin(subIndx, num, 1048576)
```

Comment:
It took me quite a while to find this issue. Why was this auto-generated value used? Have you been manually changing this for previous versions?

Reply:
This was addressed in sszgen as per https://github.com/ferranbt/fastssz/issues/127 but not sure if it has made it into a release version yet.